app/controllers/catalog_controller.rb

```diff
@@ -150,6 +150,17 @@ def catalog_item_types
     }
   end
 
+  def generic_item_subtypes
+    {
+      "custom"          => "Custom",
+      "vm"              => "VM",
+      "playbook"        => "Playbook",
+      "hosted_database" => "Hosted Database",
+      "load_balancer"   => "Load Balancer",
+      "storage"         => "Storage"
+    }
```

Comment:
@mzazrivec Thanks, we will get these updated.  This is not used yet, the first step was adding the model support and a followup PR will start displaying these values in the UI when creating a generic catalog item.